app/models/rating.rb

```diff
@@ -0,0 +1,6 @@
+class Rating < ApplicationRecord
+  validates :rate, inclusion: { in: [1, 2, 3, 4, 5] }
+
+  belongs_to :user, required: true
```

Comment:
by [this](https://bigbinary.com/blog/rails-5-makes-belong-to-association-required-by-default) article belongs_to assoc required by defailt in rails 5
```suggestion
  belongs_to :user
```